### PR TITLE
feat(react): Enable react signin at 100%

### PR DESCRIPTION
**Pull request by vpomerleau**
## Because

- We want to roll out react signin to all users

## This pull request

- Sets react signin routes to full prod rollout

## Issue that this pull request solves

Closes: #FXA-9427

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

**Review by chenba — APPROVED** [review]
💯

**Review by LZoog — APPROVED** [review]
Incredible that this didn't require extra functional test modifications, amazing job on making those signin tests compatible with Backbone and React 🎉 

Edit: will merge this before the tag on Wednesday assuming no other problems with React signin.

**Comment by LZoog** [comment]
OK, going to rebase this against `main` to ensure things are still passing, then will merge once CI goes green again. 🎉

File: packages/fxa-settings/config/webpack.config.js [hunk 1]
```diff
@@ -330,7 +330,10 @@ module.exports = function (webpackEnv) {
           'react-dom$': 'react-dom/profiling',
           'scheduler/tracing': 'scheduler/tracing-profiling',
         }),
-        fxaCryptoDeriver: require.resolve('@fxa/vendored/crypto-relier'),
+        fxaCryptoDeriver: path.resolve(
```
Comment (vbudhram) [inline packages/fxa-settings/config/webpack.config.js:333]: There is probably a better way to do this but I copied from content-server for meantime.

Comment (LZoog) [reply packages/fxa-settings/config/webpack.config.js:333]: @vbudhram NICE FIND

Okay actually I didn't mean for that to be in all caps, apparently my caps lock was on, but I'mma leave it

Comment (StaberindeZA) [reply packages/fxa-settings/config/webpack.config.js:333]: Just curious, what necessitated this change?

Comment (StaberindeZA) [reply packages/fxa-settings/config/webpack.config.js:333]: Just some background, I recently made some changes [in this PR](https://github.com/mozilla/fxa/pull/17133/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) that touches this, and was hoping a change liked this wouldn't be necessary.

**Review by LZoog — APPROVED** [review]
I'm approving again for good measure since there's been some changes. Thanks everyone!

I'm also going to go ahead and merge this. The only thing failing is one known to be flaky unrelated Cloud Scheduler test.

**Pull request merged**